state/state.go

```diff
@@ -49,6 +49,17 @@ type DynamoDBStater struct {
 }
 
 func NewDynamoDBStater(session *session.Session, service string, backfillHrs int) *DynamoDBStater {
+	svc := dynamodb.New(session)
+	input := &dynamodb.DescribeTableInput{
+		TableName: aws.String(DynamoTableName),
+	}
+	_, err := svc.DescribeTable(input)
+	if err != nil {
+		// For some reason, we cannot write to
+		// the table or access it
+		logrus.WithField("tableName", DynamoTableName).Fatal("--highavail requires an existing DynamoDB table named appropriately, please refer to the README.")
```

Comment:
Can we return `err` and let the main function error out instead of `log.Fatal` here? Generally like to try and keep exit points from the program minimal.